src/core.jsx

```diff
@@ -91,7 +92,10 @@ export default (
 
     render() {
       if (!this.isLoaded()) {
-        return <LoadingIndicator {...this.state.props} />
+        if (LoadingIndicator) {
+          return <LoadingIndicator {...this.state.props} />
+        }
+        return renderLoadingIndicator(this.state.props)
```

Comment:
You should use ```isFunction``` to ensure that ```renderLoadingIndicator``` is provided and a function before calling it 😄. Users may want to display nothing while props in ```wait``` property aren't thruthy.